tools/app_interface_reporter.py

```diff
@@ -34,9 +35,13 @@
 CONTENT_FORMAT_VERSION = '1.0.0'
 DASHDOTDB_SECRET = os.environ.get('DASHDOTDB_SECRET',
                                   'app-sre/dashdot/auth-proxy-production')
+APP_SRE_PROD_01_PROMETHEUS_URL = \
```

Comment:
You can get this URL from a graphql query instead of hardcoding it here

Reply:
Yeah, I know, I was suggested by @BumbleFeng to hardcode this for now just to simplify the process. As we know that tekton jobs only run on app-sre-prod-01.